src/puppeteer_utils.js

```diff
@@ -511,6 +512,58 @@ export const infiniteScroll = async (page, options = {}) => {
     }
 };
 
+/**
+ * Saves a full screenshot and HTML of the current page into a Key-Value store.
+ * @param {Object} page
+ *   Puppeteer <a href="https://pptr.dev/#?product=Puppeteer&show=api-class-page" target="_blank"><code>Page</code></a> object.
+ * @param {Object} [options]
+ * @param {String} [options.key=SNAPSHOT]
+ *   Key under which the screenshot and HTML will be saved. `.jpg` will be appended for screenshot and `.html` for HTML.
+ * @param {Number} [options.quality=50]
```

Comment:
for clarity, I think this should be called `screenshotQuality`, otherwise it's not clear what the quality refers to

Reply:
Fixed